CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## 11.3.21 (08/22/23)
+## 11.3.22 (08/22/23)
 
 * Fixed S3 metric name for completed multipart uploads. [#30712](https://github.com/gravitational/teleport/pull/30712)
 * Fixed Teleport Connect to properly show errors from the remote end. [#30694](https://github.com/gravitational/teleport/pull/30694)
```

Comment:
The linked PR provided a fix for Web UI, not for Connect.

It should be something like: 
_Fixed Web UI to properly show errors from the remote end in SSH sessions._

Reply:
@zmb3 pointed this out on the v13 release which I fixed there but forgot to propagate here (and unfortunately v12 - I will update the release notes on the github release, but it is now fixed in the repo). I have updated it with Zac's description: "Ensure that SSH session errors are reported to the terminal. "